### PR TITLE
#1 - Added pom.xml, .gitattributes & some additions to .gitignore

### PR DESCRIPTION
**Pull request by olegmoz**

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#3](https://github.com/artipie/nuget-adapter/issues/3) is assigned to @iakunin[/z](https://www.0crat.com/u/iakunin), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/8c4afa3c-c257-44c2-afc9-a31b7d6cc0a4); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* text=auto eol=lf
```
Comment (iakunin) [inline .gitattributes]: @olegmoz maybe it's better to add a comment like it's done [here](https://github.com/artipie/artipie/blob/master/.gitattributes#L1)?

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (iakunin) [inline pom.xml]: @olegmoz what's about copyright comment? Maybe we should add it here like it's done [here](https://github.com/artipie/artipie/blob/master/pom.xml#L2-L24), what do you think?

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.artipie</groupId>
+  <artifactId>nuget-adapter</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <name>nuget-adapter</name>
+  <url>https://github.com/artipie/nuget-adapter</url>
```
Comment (iakunin) [inline pom.xml]: @olegmoz we also can add `<description>` tag. Something trivial like `Nuget adapter for artipie`, but it's up to you

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+  </build>
+  <profiles>
+    <profile>
+      <id>qulice</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>com.qulice</groupId>
+            <artifactId>qulice-maven-plugin</artifactId>
+          </plugin>
```
Comment (iakunin) [inline pom.xml]: @olegmoz should we also specify `version` and `configuration`? Like it's done [here](https://github.com/artipie/artipie/blob/master/pom.xml#L119-L126).

File: pom.xml [hunk 5]
```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.artipie</groupId>
+  <artifactId>nuget-adapter</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <name>nuget-adapter</name>
+  <url>https://github.com/artipie/nuget-adapter</url>
+  <properties>
```
Comment (iakunin) [inline pom.xml]: @olegmoz `<license>` tag is missing (see [example](https://github.com/artipie/artipie/blob/master/pom.xml#L40-L45))

File: pom.xml (hunk 3 above)
Comment (olegmoz) [reply pom.xml]: Do not think it is needed now

**Comment by olegmoz** [comment]
Fixed all the issues

File: pom.xml [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.artipie</groupId>
+  <artifactId>nuget-adapter</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <name>nuget-adapter</name>
+  <url>https://github.com/artipie/nuget-adapter</url>
+  <licenses>
+    <license>
+      <name>MIT</name>
+      <url>https://github.com/artipie/rpm-adapter/blob/master/LICENSE.txt</url>
```
Comment (iakunin) [inline pom.xml]: @olegmoz seems that licence url leads to wrong repository

Comment (olegmoz) [reply pom.xml]: Fixed the link, thank you!

**Comment by iakunin** [comment]
@olegmoz by the way: could I ask you don't force push? Because in this case it's really hard to understand what's changed.

**Comment by 0crat** [comment]
Job was finished in 1 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Job audit: `ARC` didn't merge or close PR

**Comment by 0crat** [comment]
Order was finished: +20 point(s) just awarded to @iakunin[/z](https://www.0crat.com/u/iakunin)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Pull request merged**